packages/experiments/src/components/Button/Button.view.tsx

```diff
@@ -1,19 +1,23 @@
 /** @jsx withSlots */
 import { Stack, Text, KeytipData } from 'office-ui-fabric-react';
 import { withSlots, getSlots } from '../../Foundation';
-import { getNativeProps, buttonProperties } from '../../Utilities';
+import { getNativeProps, anchorProperties, buttonProperties } from '../../Utilities';
 import { Icon } from '../../utilities/factoryComponents';
 
 import { IButtonComponent, IButtonProps, IButtonRootElements, IButtonSlots, IButtonViewProps } from './Button.types';
 
 export const ButtonView: IButtonComponent['view'] = props => {
   const { icon, content, children, disabled, onClick, allowDisabledFocus, ariaLabel, keytipProps, buttonRef, ...rest } = props;
 
+  const rootType = _deriveRootType(props);
+  const htmlType = rootType === 'a' ? 'link' : 'button';
+  const propertiesType = rootType === 'a' ? anchorProperties : buttonProperties;
+
   // TODO: 'href' is anchor property... consider getNativeProps by root type
-  const buttonProps = { ...getNativeProps(rest, buttonProperties) };
+  const buttonProps = { ...getNativeProps(rest, propertiesType) };
```

Comment:
TS3.5 is requiring explicit types for `getNativeProps` calls so you may have to add one here. You can search any other use of `getNativeProps` for examples.

Reply:
Here's an example: https://github.com/OfficeDev/office-ui-fabric-react/blob/6ff7873e60436513ce7c3e4f39982a461fa5c86f/packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx#L72

Just swap the HTMLElement typing.